omero/sysadmins/unix/server-install-homebrew.txt

```diff
@@ -13,9 +13,9 @@ OMERO.server Mac OS X installation walk-through with Homebrew
     from the downloads page.**
 
 The instructions provided here depend on Homebrew 0.9 or later, including
-support for the ``brew tap`` command. These instructions are implemented in an
-:source:`automated script <docs/hudson/OMERO-homebrew-install.sh>` which
-installs OMERO via Homebrew from a fresh configuration.
+support for the ``brew tap`` command. These instructions are implemented in a
+series of :source:`automated scripts <https://github.com/ome/omero-install>`
```

Comment:
I think we are meeting three possibilities:
- linking to the top-level folder
- linking to the `homebrew` folder
- linking to the `install_omero5x` file
The top-level one is the more generic (and maybe less useful) but also the less fragile one. Thoughts?